vue/src/components/cms/UserList.vue

```diff
@@ -1,3 +1,5 @@
+<!-- This panel shows a list of all registered users. For each user the profil page can be visited and edited.
+     It also allows to promote, demote and delte users -->
```

Comment:
"This panel shows a list of all registered users. Each user profil can be visited directly in view- or edit-mode.
"It also allows to promote, demote and delete users"